src/main/cli/cli.c

```diff
@@ -3363,7 +3368,7 @@ static void cliRebootEx(rebootTarget_e rebootTarget)
     cliPrint("\r\nRebooting");
     bufWriterFlush(cliWriter);
     waitForSerialPortToFinishTransmitting(cliPort);
-    stopPwmAllMotors();
+    motorShutdown();
```

Comment:
Not sure about naming here - especially as the naming was confusing in the 'old world'. Is `motorShutdown()` meant to stop all motors (as opposed to letting them free wheel) and then shut down the motor signal? Should this be `motorStopAndShutdown()`?

Reply:
`motorShutdown` is an equivalent of `stopPwmAllmotors`, which creates no-signal condition. For PWM protocols, this is low-level flat signal, and for DShot over PWM, this does nothing atm (it relies on client not calling motor updates).

I think the naming would further be changed in the following refactors as we (or I) accumulate some more experience and understanding of why different places in the code call different stop / shutdown functions, like started in https://github.com/betaflight/betaflight/issues/8523#issuecomment-511275994.

I want to keep it as is for the moment and get refactoring rolling.

Reply:
Sounds good to me - can you rebase / squash please?